static/js/modules/filter-panel.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+var FilterSet = require('./filter-set').FilterSet;
+var accessibility = require('fec-style/js/accessibility');
+
+var defaultOptions = {
+  body: '.filters',
+  form: '#category-filters',
+  toggle: '#filter-toggle'
+};
+
+function FilterPanel(options) {
+  this.isOpen = false;
+  this.options = _.extend({}, defaultOptions, options);
+
+  this.$body = $(this.options.body);
+  this.$form = $(this.options.form);
+  this.$toggle = $(this.options.toggle);
+
+  this.$toggle.on('click', this.toggle.bind(this));
+
+  this.filterSet = new FilterSet(this.$form).activate();
+  if (!_.isEmpty(this.filterSet.serialize())) {
+    this.show();
+  }
+
+  this.adjust();
+}
+
+FilterPanel.prototype.adjust = function() {
+  if ($('body').width() > 768) {
```

Comment:
What does the number 768 represent? Is it a breakpoint that's shared with the CSS?